docs/af-fundamentals-tutorial/modules/dlq/pages/index.adoc

```diff
@@ -0,0 +1,15 @@
+= Dead Letter Queues in Axon Framework
+:navtitle: Dead letter queues
+
+== Introduction
+
+Once you are running a big application with lots of events in production, things might go wrong.
+This could just be temporary because an external API is down, or it may require changing the code when there is a bug with a combination of not-tested events.
+
+The default configuration for event processors is just to log errors, likely in production you already changed it, to throw the error instead.
+However, this can cause an event processor to stop processing events, as it keeps trying to process the same event until it stops causing an exception.
+
+To prevent this it's possible to configure the use of a sequenced dead letter queue (DLQ). With DLQ instead of retrying, the framework places any event throwing an error in a queue. To keep the order of the events correct, the framework also places all further events having the same sequence identifier _(by default, the sequence identifier is the aggregate id)_ directly into the queue. Such an approach assures all events for the same aggregate, have to wait on the event causing an exception, while the event processor doesn't consume resources continuously trying to process unprocessable events.
```

Comment:
```suggestion
To prevent this it's possible to configure the use of a sequenced dead letter queue (DLQ). With a DLQ, instead of retrying, the framework places any event throwing an error in a queue. To keep the order of the events correct, the framework also places all further events having the same sequence identifier _(by default, the sequence identifier is the aggregate id)_ directly into the queue. Such an approach assures all events for the same aggregate, have to wait on the event causing an exception, while the event processor doesn't consume resources continuously trying to process unprocessable events.
```